tests/append-empty-mock/package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
is this file needed? I don't see how it is used.

Reply:
Not really sure tbh, just made a copy of `tests/append-mock`

Reply:
ah ok, no problem, just wondering if I missed something. Thanks for contributing! :+1: